src/main/java/dev/devature/penguin_api/controller/IssueController.java

```diff
@@ -0,0 +1,97 @@
+package dev.devature.penguin_api.controller;
+
+import dev.devature.penguin_api.entity.Issue;
+import dev.devature.penguin_api.enums.IssueResult;
+import dev.devature.penguin_api.service.IssueService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/issue")
+public class IssueController {
+
+    private final IssueService issueService;
+
+    @Autowired
+    public IssueController(IssueService issueService){
+        this.issueService = issueService;
+    }
+
+    /**
+     * @param issue Take in a RequestBody with the issue object created.
+     * @return ResponseEntity with status and body based on the result of service.
+     */
+    @PostMapping("/new")
```

Comment:
We can do a PostMapping to the base issue URL here

```suggestion
    @PostMapping
```